### PR TITLE
Finished static_handler

### PR DESCRIPTION
**Pull request by Filisha**

### CODE REVIEW

File: webserver.dSYM/Contents/Info.plist [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+		<key>CFBundlePackageType</key>
+		<string>dSYM</string>
+		<key>CFBundleSignature</key>
+		<string>????</string>
+		<key>CFBundleShortVersionString</key>
+		<string>1.0</string>
+		<key>CFBundleVersion</key>
+		<string>1</string>
+	</dict>
+</plist>
```
Comment (evangrayk) [inline webserver.dSYM/Contents/Info.plist]: Can you delete `webserver` and the *.dSYM stuff?

File: src/static_handler.cc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        if(config.statements_[i]->tokens_[0] == "root"){
+          filebase_ = config.statements_[i]->tokens_[1];
+        }
+        else{
+          std::cerr << "Invalid Static Handler 'filebase' parameter." << std::endl;
+          return RequestHandler::Error;
+        }
+    }
+    else{
+      std::cerr << "Static Handler must specify 'filebase' parameter." << std::endl;
```
Comment (evangrayk) [inline src/static_handler.cc]: I think this error message is inaccurate? Maybe should say "invalid Static Handler config" or something

File: webserver.dSYM/Contents/Info.plist (hunk 1 above)
Comment (joeydlee95) [reply webserver.dSYM/Contents/Info.plist]: *.dSYM should not be merged with master ... same with webserver

File: src/static_handler.cc (hunk 2 above)
Comment (joeydlee95) [reply src/static_handler.cc]: In the future is it possible to have like invalid << request.getHandler() << ".\n"

Comment (Filisha) [reply src/static_handler.cc]: Will do @evangrayk 
@joeydlee95  sure take the lead

**Pull request merged**